dss/events/handlers/index.py

```diff
@@ -50,33 +43,41 @@ def is_bundle_to_index(key) -> bool:
     return result is not None
 
 
-# TODO Future - Identified in the spec yet not needed for current prototype.
-def check_for_tombstone(bucket, key, log):
-    pass
-
-
-def read_bundle_manifest(s3, bucket, bundle_key, log):
-    manifest_string = s3.Object(os.getenv("DSS_S3_TEST_BUCKET"), bundle_key).get()['Body'].read().decode("utf-8")
-    log.debug("Read bundle manifest from bucket %s with bundle key %s: %s", bucket.name, bundle_key, manifest_string)
+def read_bundle_manifest(s3, bucket_name, bundle_key, logger):
+    manifest_string = s3.Object(bucket_name, bundle_key).get()['Body'].read().decode("utf-8")
+    logger.debug("Read bundle manifest from bucket %s with bundle key %s: %s", bucket_name, bundle_key, manifest_string)
     manifest = json.loads(manifest_string, encoding="utf-8")
     return manifest
 
 
-def create_index_data(s3, bucket, bundle_key, manifest, log):
+def create_index_data(s3, bucket_name, manifest, logger):
     index = dict(state="new", manifest=manifest)
     files_info = manifest['files']
     index_files = {}
-    bucket = s3.Bucket(os.getenv("DSS_S3_TEST_BUCKET"))
+    bucket = s3.Bucket(bucket_name)
     for file_info in files_info:
         if file_info['indexed'] is True:
-            if not file_info["name"].endswith(".json"):
-                log.warning("File %s is marked for indexing but is not of type JSON. It will not be indexed.")
+            if not file_info["name"].endswith(".json"):  # TODO Don't check filename, if file is 'indexed' load json
+                logger.warning("File %s is marked for indexing but is not of type JSON. It will not be indexed.",
```

Comment:
Thanks, that is more clear.
I have work in progress elsewhere (outside of this PR) to change to the approach of parsing the file as JSON rather than relying on its suffix, and unit test accordingly.